[APPSEC] Add docs for the ASM Threat Overview page (#21924)
DataDog/documentation: content/en/security/application_security/threats/threat-overview.md

---
title: Threat Overview
kind: documentation
aliases:  
---

{{< img src="security/application_security/threats/appsec-threat-overview-page-top.png" alt="Screenshot of the ASM Threat Overview page"  >}}

## Introduction

> The ASM Threats Overview provides an at a glance view of your application and API posture, highlighting trends, service exposure, attack traffic, and the impact on business logic. Users can quickly pivot from widgets to related traces.

Reviewer: `Threat Overview` is the proper name

> The ASM Threats Overview provides an at a glance view of your application and API posture, highlighting trends, service exposure, attack traffic, and the impact on business logic. Users can quickly pivot from widgets to related traces.

Author: Incorporating suggestion and replacing all `Threats` to `Threat`

The Threats Overview is divided into several sections, each focusing on a different aspect of security with supporting information.

> ## Sections

Reviewer 2: Consider formatting these sections as a description list. I think it would look nice and improve readability. See https://datadoghq.atlassian.net/wiki/spaces/docs4docs/pages/1182958054/Markdown#Description-lists for the Markdown and an example.

> **Attack Surface Area**: This section provides insights into the exposed services, the tools attackers are using, and the commercial scanners that are identifying potential vulnerabilities.

Reviewer 2: ```suggestion
**Attack Surface Area**: This section provides insights into the exposed services, the tools attackers are using, and the commercial scanners that identify potential vulnerabilities.
```

> **Attack Traffic**: These graphs identify the classification of attacks, such as SSRF, LFI, SQL and command injection, allowing users to identify spikes in malicious traffic and patterns.

Reviewer 2: ```suggestion
**Attack Traffic**: These graphs identify the classification of attacks, such as SSRF, LFI, SQL and command injection. They allow users to identify spikes in malicious traffic and patterns.
```

> **Business Logic**: This section focuses on the application-specific operations, such as account takeover attempts and custom business logic metrics, to help you understand trending attacks related to user accounts or fraud.

Reviewer 3: Maybe we can cut it shorter like this?
**Business Logic**: This section focuses on fraud and business logic abuse such as account takeover attempts or any custom business logic events tracked by your application.

Reviewer: +2

> **Business Logic**: This section focuses on the application-specific operations, such as account takeover attempts and custom business logic metrics, to help you understand trending attacks related to user accounts or fraud.

Reviewer 2: ```suggestion
**Business Logic**: This section focuses on application-specific operations, such as account takeover attempts and custom business logic metrics. Trending attacks related to user accounts or fraud appear in this section.
```

> **Geographic Overview**: A global heatmap indicating the sources of attack traffic, providing a visual representation of threats by region.

Reviewer 2: ```suggestion
**Geographic Overview**: A global heat map indicating the sources of attack traffic, providing a visual representation of threats by region.
```

## Best Practices

1. Users should review trends and adopt a corresponding protection policy to meet their posture needs.
2. Regularly review the Exposed Services widget to ensure only intended services are accessible and that they have a protection policy that meets your risk profile.
3. Block attack tools and ensure that customer scanners are part of an authorized vulnerability management program.

> 4. Monitor business logic for changes in detection profiles related to distributed credential stuffing campaigns or risky payment activity.

Reviewer 3: Maybe what I put below is too simplistic but `changes in detection profiles` is a blurry notion for me.
4. Motinor business logic for spikes in credential stuffing attacks or risky payment activity

Reviewer: +2

> 5. Use the geographic overview to compare the attack traffic sources with your expected customer sources.

Reviewer 3: I would replace `sources` with a wording like `locations` maybe?

Reviewer: +2
